README.md

```diff
@@ -12,7 +12,6 @@ This action enables the integration of PagerDuty into StackStorm. It is capable
 Copy the example configuration in [pagerduty.yaml.example](./pagerduty.yaml.example)
 to `/opt/stackstorm/configs/pagerduty.yaml` and edit as required.
 
-* `subdomain:` name of subdomain
 * `api_key:` API-KEY
 * `service_api:` SERVICE-API
 * `debug:` optional debug flag. Set to True for additional logging
```

Comment:
Yes. I've been running `st2 pack config pagerduty` to create the config file. I'll make note of the `st2ctl reload --register-configs`.